.github/workflows/release.yml

```diff
@@ -53,6 +58,7 @@ jobs:
           export QUARKUS_CONTAINER_IMAGE_PASSWORD="${{ secrets.IMAGE_REPO_PASSWORD }}"
           export QUARKUS_CONTAINER_IMAGE_PUSH="true"
           export QUARKUS_CONTAINER_IMAGE_ADDITIONAL_TAGS=latest
+          export QUARKUS_CONTAINER_IMAGE_TAG="${{ env.PROJECT_VERSION }}"
```

Comment:
I think exporting here for the release workflow won't work because the POM file still has the `-SNAPSHOT` version, but running the `release:perform` will build the non-SNAPSHOT. However, I think it is not an issue because there is nothing to actually change for the non-SNAPSHOT, so perhaps the release job doesn't need to change at all (as long as we stick with x.y.z releases).

Reply:
In that case i'll remove the changes. Thanks